pkg/didcomm/dispatcher/api.go

```diff
@@ -47,6 +47,8 @@ type StateMsg struct {
 	Type    StateMsgType
 	StateID string
 	Msg     DIDCommMsg
+	// Properties contains specific keys from different protocols
+	Properties map[string]interface{}
```

Comment:
Suggest we rename this to "Context". The godoc is misleading - in the context of Aries, "keys" usually means "cryptographic keys". Suggest we make this a bit more clear.

Reply:
+1 on using a term other than keys for the reason given by @llorllale .

I'm not sure, though, that we should use the term context in yet another way :).

Reply:
I change the comments

Reply:
> I'm not sure, though, that we should use the term context in yet another way :).

I was expecting this counterpoint, but this is really what it is. :)

As a matter of fact, we might want to consider using the actual `context.Context` in the future. The timeout/deadline functionality would align nicely with the `~timing` decorator, and with protocol-specific K-Vs stuffed into it, just like in this case.